packages/integration-sdk-runtime/src/storage/memory.test.ts

```diff
@@ -0,0 +1,214 @@
+import {
+  Entity,
+  ExplicitRelationship,
+  Relationship,
+} from '@jupiterone/integration-sdk-core';
+import { InMemoryGraphObjectStore } from './memory';
+import { v4 as uuid } from 'uuid';
+
+function createTestEntity(partial?: Partial<Entity>): Entity {
```

Comment:
You pulled these out earlier in this PR  to `packages/integration-sdk-private-test-utils/src/graphObject.ts`

Reply:
Good call. Fixing.